.pullapprove.yml

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
I'm new to pullapprove and accidentally changed the file.
I'll redo the pull request